testador/zzipinternet.sh

```diff
@@ -1 +1,18 @@
-$ zzipinternet   #=> --regex ^(([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.{0,1}){4}$
+# Testa IPV4
+$ zzipinternet    #=> --regex (^$|^(([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])$)
+$ zzipinternet -4 #=> --regex (^$|^(([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])$)
```

Comment:
```suggestion
$ zzipinternet -4 #=> --regex (^$|^[0-9.]+$)
```

Como agora temos as `zztool testa_ipv[46]` que fazem o match bem preciso do IP retornado, acho que aqui nesses testes da `zzipinternet` poderíamos relaxar e usar regex mais simples, genéricas.

Reply:
pode ser. Tive que alterar pois a regex original (que já era da versão anterior do zzipinternet aceitava um ip acabando em ponto, e como o ponto era sempre opcional, aceitava `127.1` como ip válido. Mas já que está arrumado na zztool, aqui está tranquilo!